network_modeling/models/cray-network-hardware.yaml

```diff
@@ -150,13 +150,23 @@ network_hardware:
     ports:
       - count: 2
         speed: 10
-  - name: "Customer Edge Router - generic stub"
-    vendor: "none"
+  - name: "arista_edge_router"
+    vendor: "arista"
     type: "switch"
     model: "customer_edge_router"
     ports:
       - count: 48
         speed: 100
+  - name: "aruba_jl636a_edge_router"
```

Comment:
Might want to use the same naming convention as the other aruba switches `- name: "Aruba 8360 JL706A"` for example.